qfieldsync/core/cloud_api.py

```diff
@@ -133,9 +201,21 @@ def __init__(self, parent=None) -> None:
         self.is_login_active = False
 
         url = self.preferences.value("qfieldCloudServerUrl")
+
         # we should always use the QgsNetworkAccessManager instance, otherwise ssl handling is impossible
         self._nam = QgsNetworkAccessManager.instance()
 
+        self.auth_method = CloudAuthMethod(
+            self.preferences.value("qfieldCloudAuthMethod")
+        )
+        pref_auth_config_id = self.preferences.value("qfieldCloudAuthcfg")
+        if pref_auth_config_id:
+            auth_config = QgsAuthMethodConfig()
+            _, config = QgsApplication.authManager().loadAuthenticationConfig(
```

Comment:
```suggestion
            _whatever, config = QgsApplication.authManager().loadAuthenticationConfig(
```

I find it nice when one names unused variables, so the reader does need to check the API docs. In this case `whatever` should be changed with whatever the value of the variable represents.